index.js

```diff
@@ -149,7 +155,15 @@ oauthPlugin.APPLE_CONFIGURATION = {
   authorizeHost: 'https://appleid.apple.com',
   authorizePath: '/auth/authorize',
   tokenHost: 'https://appleid.apple.com',
-  tokenPath: '/auth/token'
+  tokenPath: '/auth/token',
+  [kGenerateCallbackUriParams]: function (callbackUriParams, requestObject, scope, state) {
```

Comment:
Just so I understand you want to use a symbol here so that it can't collide with any other parameters that other oauth implementations might have added over time? I think that serves the purpose but is potentially a little confusing for anyone reading this code later. I feel like it might be simpler to pick a parameter name that makes sense to you and then do a Github search or something like that to build confidence no one else is using it -- I feel like there aren't *that* many other oauths out there such that you could probably pick a unique name. Up to you though!